app/controllers/v0/triage_teams_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module V0
+  class TriageTeamsController < HealthcareMessagingController
+    def index
+      @teams = client.get_triage_teams
```

Comment:
I was using it in a controller test, but that is kind of redundant now with the request spec - I can make it local.